components/ide/jetbrains/gateway-plugin/src/main/kotlin/io/gitpod/jetbrains/gateway/GitpodConnectionProvider.kt

```diff
@@ -444,3 +478,11 @@ class GitpodConnectionProvider : GatewayConnectionProvider {
     private data class SSHHostKey(val type: String, val hostKey: String)
 
 }
+
+fun ThinClientHandle.prettyPrint(): String {
+    return "uid=${uid}, clientPresent=${clientPresent}, lifeTime={status=${lifetime.status}, alive=${lifetime.isAlive}}"
+}
+
+fun Logger.d(message: String) {
```

Comment:
Make the logging statement an extension function to make it easier for log filtering, and also it would be easier if we need to cleanup someday (IDEA will help us finding all the referenced position)